doc/classes/FogVolume.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="FogVolume" inherits="VisualInstance3D" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		A node used to add local fog with the volumetric fog effect.
+		A region that overrides the default volumetric fog of the world environment.
```

Comment:
I think "contributes" is the better word. Saying "overrides" sounds like it just replaces the values. But it doesn't, it has access to the existing values and modifies them. Sure, it can completely override them, but that's just a special case of contributing to it. So the word "contributes" implies that it can also override, while "overrides" does not make it clear that it doesn't need to override it.

Reply:
So "modifies" then?

Reply:
The way  volumetric fog works is:
1. The contribution from each fog volume is added together (no one volume overrides another, they are all added together)
2. The contribution from all fog volumes is added to the default fog from the WorldEnvironment (the global density)

At no point does one fog type override another as everything is additive (or subtractive because negative densities are allowed)

Reply:
Alrighty, I actually pushed this already